### DISCUSSION TITLE
Can I have parcel generate script tag without defer?

### DISCUSSION THREAD

**Discussion by Kasparsu** [parcel-bundler/parcel#7699] [Q&A] [answered]
Can I have parcel generate script tag without defer? Looked around and haven't found an easy answer. I have a crappy webxr library that well goes bananas when defer is used.

**Comment by andre-krueger** [comment, marked as answer]
+1
Have an [issue](https://github.com/bigskysoftware/htmx/issues/887) because of this with htmx myself.
I need to use `type="module"` to be able to import htmx and one of its extensions, but then defer is added to the script tag, which results in the htmx extension not working, because it's trying to add a function to the htmx instance.
When I remove the defer attribute from the bundled html, the extension works.

Edit:
Just for info, I solved my issue with this code:
```
<script src="npm:htmx.org"></script>
<script src="npm:htmx.org/dist/ext/preload.js"></script>
```
